services/src/messaging/src/test/java/org/openkilda/messaging/nbtopology/request/LinkPropsDropTest.java

```diff
@@ -27,20 +27,21 @@
 public class LinkPropsDropTest implements StringSerializer {
     @Test
     public void serializeLoop() throws Exception {
-        LinkPropsDrop origin = makeRequest();
+        org.openkilda.messaging.nbtopology.request.LinkPropsDrop origin = makeRequest();
```

Comment:
You can leave the previous version. Hereinafter you can use just `LinkPropsDrop` without packages.

Reply:
Fixed.